.github/CODEOWNERS

```diff
@@ -257,7 +257,7 @@ pkgs/development/python-modules/buildcatrust/ @ajs124 @lukegb @mweinelt
 # PHP interpreter, packages, extensions, tests and documentation
 /doc/languages-frameworks/php.section.md          @aanderse @drupol @etu @globin @ma27 @talyz
 /nixos/tests/php                                  @aanderse @drupol @etu @globin @ma27 @talyz
-/pkgs/build-support/build-pecl.nix                @aanderse @drupol @etu @globin @ma27 @talyz
+/pkgs/build-support/php                           @aanderse @drupol @etu @globin @ma27 @talyz
```

Comment:
Are all maintainers still on board for this?

Reply:
I'm waiting for feedback, but I believe they are busy... this is the reason why this PR took so long. As you said, maybe it's better to merge it and do modifications afterwards.

Reply:
Please keep the old line or migrate the old line to the new build-pecl.nix, create a new one and add yourself to this one alone or with the consent of the remaining maintainers.

So we move on regarding this.

Reply:
I'm afraid this is going to be quite cumbersome. Creating that directory is essential since it contains new files required for the new builder.
This PR has been opened since April and I **guess** most of the maintainers are OK with this; if this was something sensitive, they would have raised their voice immediately.

Reply:
I implemented the changes after discussing on Matrix. I implemented the changes.